### PR TITLE
Refunds invoices

### PR DESCRIPTION
**Pull request by mlantz**
Now you can refund a user

### CODE REVIEW

File: src/Billable.php [hunk 1]
```diff
@@ -184,13 +201,11 @@ public function invoice()
...
         if ($this->stripe_id) {
             try {
-                StripeInvoice::create(['customer' => $this->stripe_id], $this->getStripeKey())->pay();
+                return StripeInvoice::create(['customer' => $this->stripe_id], $this->getStripeKey())->pay();
             } catch (StripeErrorInvalidRequest $e) {
                 return false;
             }
         }
-
-        return true;
```
Comment (GrahamCampbell) [inline src/Billable.php]: this refactor is incorrect

Comment (mlantz) [inline src/Billable.php]: Suggestions? I'd like to be able to get back the invoice object from Stripe, and returning the object from Stripe would not be considered false.

**Comment by mlantz** [comment]
Didn't realize what I had done. Better?

File: src/Billable.php [hunk 2]
```diff
@@ -52,6 +53,22 @@ public function charge($amount, array $options = [])
     }
 
     /**
+     * Refund a customer for a charge.
+     *
+     * @param  str  $charge
```
Comment (GrahamCampbell) [inline src/Billable.php]: string, not str

**Pull request merged**